src/components/Garden/Feed/HeroBanner.js

```diff
@@ -22,15 +28,29 @@ const BANNERS = {
   max: { image: desktopBanner, aspectRatio: '159%' },
 }
 
+const BANNERS_DARK_MODE = {
+  small: { image: mobileBannerDark, aspectRatio: '54%' },
+  medium: { image: tabletBannerDark, aspectRatio: '36%' },
+  medium_full: { image: tabletBannerFullDark, aspectRatio: '36%' },
+  large: { image: desktopBannerDark, aspectRatio: '159%' },
+  max: { image: desktopBannerDark, aspectRatio: '159%' },
+}
+
 function HeroBanner({ onRequestNewProposal, fullScreenMode }) {
   const { account } = useWallet()
   const { layoutName } = useLayout()
+  const { appearance } = useAppTheme()
 
   const mobileMode = layoutName === 'small'
   const tabletMode = layoutName === 'medium'
   const compactMode = mobileMode || tabletMode
+
+  const bannerType = appearance === 'light' ? BANNERS : BANNERS_DARK_MODE
+
   const banner =
-    fullScreenMode && tabletMode ? BANNERS.medium_full : BANNERS[layoutName]
```

Comment:
No esta en Staging el fullScreenmMode

Reply:
Seems like something we added for handling [some error on no byot orgs](https://github.com/1Hive/gardens-ui/commit/3ec12cdeaf7c6032e1ef7177149570d12c72afd6)

Currently it is not implemented on staging.